### ISSUE TITLE
Conflict with Workona?

### ISSUE THREAD

**Issue by pixel8mb** [Drive4ik/simple-tab-groups#877]
Can’t get to overall settings, and “manage groups” disappeared. I don’t have the main page with General, Hot Keys, and Backup tabs. I wonder if the Workona extension is a conflict? I got “Manage Groups” back, and I see my Workona tab listed in “other tabs,” not grayed out, although I put it in a group. Is that causing my problem? I’ll get rid of it if so, but really hate very much to do that if it’s not what’s causing the problem. Can you please help? Love STG but it’s not quite right now.
I’m on Windows 10, using Firefox v.94.0.1 (64-bit)
I added Simple Tab Groups on 11/4/21, must be latest version.

**Comment by pixel8mb** [comment]
PS I'm loving STG, thanks for this awesome add-on! Also note, I spent last two days organizing my many open tabs into STG groups, will really appreciate help you can provide.

**Comment by pixel8mb** [comment]
Sorry for deluge, but unless you know if Workona causes a conflict I think I found a simple answer, from screenshots I saw list of groups on each group shows "Manage groups" and "Setting" on the bottom. I had one too many groups in my list for that to show, and it didn't scroll down on my screen at least. I got rid of one group and now see those two things at the bottom. This caused me a lot searching, with worry that I might have to undo all I'd done. Should you include instructions with some more simple tips? I'm a fan still but many new users might appreciate that. Many thanks for STG. :)
